api_tests/nodes/views/test_node_files_list.py

```diff
@@ -614,15 +615,21 @@ def add_github(self):
             oauth_settings._id: []}
         addon.user_settings.save()
 
-    def check_file_order(self, resp):
-        previous_file_name = 0
-        for file in resp.json['data']:
-            int_file_name = int(file['attributes']['name'])
-            assert int_file_name > previous_file_name, 'Files were not in order'
-            previous_file_name = int_file_name
+    def check_file_order(self, resp, key, compare_func, ascending=False):
```

Comment:
`compare_func` -> `key` (for consistency with Python STL compare/sorting funcs)?